schema.sql

```diff
@@ -5,4 +5,6 @@ CREATE TABLE animals (
     escape_attempts INTEGER,
     neutered BOOLEAN,
     weight_kg DECIMAL(4, 2)
-);
\ No newline at end of file
+);
+
+alter table animals add species VARCHAR(250);
```

Comment:
* According to the [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md#:~:text=Create%20a%20table,the%20owners%20table) your project are lacking all these requirements Kindly add all of them

> Create a table named owners with the following columns:

> id: integer (set it as autoincremented PRIMARY KEY)
> full_name: string
> age: integer
> Create a table named species with the following columns:
> id: integer (set it as autoincremented PRIMARY KEY)
> name: string
> Modify animals table:
> Make sure that id is set as autoincremented PRIMARY KEY
> Remove column species
> Add column species_id which is a foreign key referencing species table
> Add column owner_id which is a foreign key referencing the owners table